plugins/inputs/mesos/README.md

```diff
@@ -173,6 +189,40 @@ Mesos master metric groups
     - registrar/state_store_ms/p999
     - registrar/state_store_ms/p9999
 
+- allocator
+    - allocator/mesos/allocation_run_ms
```

Comment:
I wonder if we should remove the second `mesos` here? The resulting Prometheus metric looks like `mesos_allocator_mesos_allocation_run_latency_ms` - maybe `mesos_allocator_allocation_run_latency_ms` would be cleaner?

Reply:
(could there be a non-mesos allocator, I wonder?)